detection-rules/link_multistage_microsoft_forms.yml

```
@@ -7,39 +7,11 @@ source: |
   and any(filter(body.links, .href_url.domain.domain == "forms.office.com"),
           // avoid doing Link Analysis if the display-text has strong indications of phishing
           (
-            any([
-                  "REVIEW",
-                  "PROPOSAL",
-                  "DOCUMENT",
-                  "PASSWORD",
-                  "eFax",
-                  "Restore",
-                  "OPEN",
-                  "0PEN",
-                  "Secure",
-                  "Messaging",
-                  "Reset",
-                  "Account",
-                  "Verify",
-                  "Login",
-                  "Notification",
-                  "Alert",
-                  "Urgent",
-                  "Immediate",
-                  "Access",
-                  "Support",
-                  "Update",
-                  "Download",
-                  "Attachment",
-                  "Service",
-                  "Payment",
-                  "Remittance",
-                  "Invoice"
-                ],
-                // replace confusables - observed ITW
-                strings.icontains(strings.replace_confusables(..display_text), .)
-            // add confidence to these strings by using profile.by_sender()
+            // replace confusables - observed ITW
+            regex.icontains(strings.replace_confusables(.display_text),
+                            'review|proposal|document|efax|restore|(o0)pen|secure|messaging|reset|account|verify|login|notification|alert|urgent|immediate|access|support|\bupdate\b|download|attachment|service|payment|remitttance|invoice'
```

- char class instead of group
- an extra `t` in remittance
```suggestion
                            'review|proposal|document|efax|restore|[o0]pen|secure|messaging|reset|account|verify|login|notification|alert|urgent|immediate|access|support|\bupdate\b|download|attachment|service|payment|remittance|invoice'
```